### PR TITLE
Fix Bug 1209455 - Error 404 for all Thunderbird release notes URLs on mozilla.org

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

File: bedrock/thunderbird/urls.py [hunk 1]
```diff
@@ -31,20 +34,26 @@
...
     url('^thunderbird/(?:%s/)?system-requirements/$' % channel_re,
         bedrock.releasenotes.views.latest_sysreq,
-        {'product': 'Thunderbird'}, name='thunderbird.sysreq'),
+        {'product': 'thunderbird'}, name='thunderbird.latest.sysreq'),
     url('^thunderbird/latest/system-requirements/$',
         bedrock.releasenotes.views.latest_sysreq,
         {'product': 'thunderbird', 'channel': 'release'}),
+    url(sysreq_re,
+        bedrock.releasenotes.views.system_requirements,
+        {'product': 'Thunderbird'}, name='thunderbird.sysreq'),
```
Comment (jgmize) [inline bedrock/thunderbird/urls.py]: Shouldn't 'Thunderbird' be lower case here?

Comment (kyoshino) [inline bedrock/thunderbird/urls.py]: No, this one is upper-case somehow. I think everything should be lower case, but it likely requires change to Nucleus.

Comment (kyoshino) [inline bedrock/thunderbird/urls.py]: I'll comment on the mixture of lower- and upper-cases.

Comment (kyoshino) [inline bedrock/thunderbird/urls.py]: Comment added. I'm happy to send a separate PR to fix the inconsistency, as `firefox/urls.py` has the same issue.

Comment (jgmize) [inline bedrock/thunderbird/urls.py]: > Comment added. I'm happy to send a separate PR to fix the inconsistency, as firefox/urls.py has the same issue.

Thanks, I agree that shouldn't block this.
